integrations/sentry/.port/resources/blueprints.json

```diff
@@ -1,4 +1,103 @@
 [
+  {
+    "identifier": "sentryUser",
+    "description": "This blueprint represents a Sentry user in our software catalog.",
+    "title": "Sentry User",
+    "icon": "Sentry",
+    "schema": {
+      "properties": {
+        "username": {
+          "type": "string",
+          "title": "Username"
+        },
+        "isActive": {
+          "type": "boolean",
+          "title": "Is Active"
+        },
+        "dateJoined": {
+          "type": "string",
+          "format": "date-time",
+          "title": "Date Joined"
+        },
+        "lastLogin": {
+          "type": "string",
+          "format": "date-time",
+          "title": "Last Login"
+        },
+        "orgRole": {
+          "icon": "DefaultProperty",
+          "title": "Organization Role",
+          "type": "string",
+          "enum": [
+            "member",
+            "admin",
+            "owner",
+            "manager",
+            "biling"
+          ],
```

Comment:
is that something static by sentry or can be adjusted by each organization?

Reply:
this is static from the API and consistent with the Sentry UI